<launch>

<include file="$(find usb_cam)/launch/usb_cam-test.launch" />

<node name="optic_flow_node" pkg="optic_flow_node" type="wide_field_optic_flow_node" output="screen" >
	<!-- EC Corridor params
        <param name="image_center_x" value="290"/>
	<param name="image_center_y" value="240"/>
	<param name="inner_ring_radius" value="220"/>
	<param name="ring_dr" value="10"/>
	<param name="num_ring_points" value="80"/>
	<param name="num_rings" value="5"/> 
	<param name="pixel_scale" value="64"/>
	-->
        <param name="image_center_x" value="288"/>
        <param name="image_center_y" value="240"/>
        <param name="inner_ring_radius" value="210"/>
        <param name="ring_dr" value="10"/>
        <param name="num_ring_points" value="80"/>
        <param name="num_rings" value="5"/>
        <param name="pixel_scale" value="36"/>

</node>

<include file="$(find goggles)/launch/goggles.launch" />

<!--
 <include file="/opt/ros/melodic/share/vrpn_client_ros/launch/optitrack.launch" />

<node pkg="vrpn_velocity" type="vrpn_vel_calc.py" name="vrpn_vel_calc" /> 
-->

<node name="simple_nearness" pkg="nearness_estimation" type="simple_nearness_node" output="screen" >
	<param name="num_ring_points" value="80"/>
</node>

<include file="$(find rplidar_ros)/launch/rplidar_a3.launch" />

</launch>
